data/collector.go

```diff
@@ -285,10 +338,60 @@ func (c *collector) writeCaptureResults() {
 				c.logger.Error(errors.Wrap(err, fmt.Sprintf("failed to write to collector %s", c.target.Path())).Error())
 				return
 			}
+
+			c.maybeWriteToMongo(msg)
 		}
 	}
 }
 
+// TabularData is a denormalized sensor reading.
+type TabularData struct {
+	TimeRequested time.Time `bson:"time_requested"`
+	TimeReceived  time.Time `bson:"time_received"`
+	ComponentName string    `bson:"component_name"`
+	ComponentType string    `bson:"component_type"`
+	MethodName    string    `bson:"method_name"`
```

Comment:
Something I'm wondering about (from the product perspective) is whether it's important that this data is queryable via the exact same queries that my application uses to query TabularDataByMQL. If so, we'd need part_id, robot_id, org_id, location_id, etc.

Reply:
I'll check with Eliot

Reply:
We decided in the scope that it's okay to omit these fields